Sources/StytchUI/Extensions/UIFont+IBMPlexSans.swift

```diff
@@ -2,59 +2,59 @@ import Foundation
 import UIKit
 
 public extension UIFont {
-    static func IBMPlexSansRegular(size: CGFloat) -> UIFont? {
-        UIFont(name: "IBMPlexSans-Regular", size: size)
+    static func IBMPlexSansRegular(size: CGFloat) -> UIFont {
+        UIFont(name: "IBMPlexSans-Regular", size: size) ?? UIFont.systemFont(ofSize: size, weight: .regular)
     }
```

Comment:
I made all the fonts have a fall back to their system equivalent so that I could have this return a non optional type, made usage here so much easier. For the italic fonts I was not able to do that so they default to their non Italic equivalent.